### PR TITLE
Add Chibisafe

### PR DESCRIPTION
**Pull request by kzshantonu**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [ ] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/chibisafe.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: "Chibisafe"
+website_url: "https://chibisafe.moe"
+source_code_url: "https://github.com/chibisafe/chibisafe"
+description: "Blazing fast file uploader and awesome bunker written in node!"
+licenses:
+  - MIT
+platforms:
+  - JavaScript
+  - Nodejs
+  - Docker
```
Comment (Ki-er) [inline software/chibisafe.yml:10]: JS isnt needed as Nodejs is already present / changed order to match docs
```suggestion
platforms:
  - Docker
  - Nodejs
```

File: software/chibisafe.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+  - File Transfer - Single-click & Drag-n-drop Upload
+depends_3rdparty: false
```
Comment (Ki-er) [inline software/chibisafe.yml:13]: Optional so can be removed.
```suggestion
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Couple changes.

File: software/chibisafe.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: "Chibisafe"
+website_url: "https://chibisafe.moe"
+source_code_url: "https://github.com/chibisafe/chibisafe"
+description: "Blazing fast file uploader and awesome bunker written in node!"
```
Comment (Ki-er) [inline software/chibisafe.yml:4]:
```suggestion
description: "Blazing fast file uploader and awesome bunker written in node."
```

File: software/chibisafe.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+name: "Chibisafe"
+website_url: "https://chibisafe.moe"
+source_code_url: "https://github.com/chibisafe/chibisafe"
+description: "Blazing fast file uploader and awesome bunker written in node."
```
Comment (nodiscc) [inline software/chibisafe.yml:4]:
```suggestion
description: "File uploader service that aims to to be easy to use and set up. It accepts files, photos, documents, anything you imagine and gives you back a shareable link for you to send to others."
```

**Review by nodiscc — CHANGES_REQUESTED** [review]
Better description

**Review by Rabenherz112 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**